packages/components/src/base-control/index.tsx

```diff
@@ -104,10 +104,10 @@ export const BaseControl = ( {
  * 	</BaseControl>
  * );
  */
-export const VisualLabel: FunctionComponent< BaseControlVisualLabelProps > = ( {
+export const VisualLabel = ( {
 	className,
 	children,
-} ) => {
+}: WordPressComponentProps< BaseControlVisualLabelProps, 'span' > ) => {
```

Comment:
By using `WordPressComponentProps`, there's no need to specify `className` in `BaseControlVisualLabelProps`


Also, curious about why you decided to switch to using `WordPressComponentProps` only for `VisualLabel` and not for `BaseControl` in the same file?

Reply:
Good catch, fixed in 46f4d9e.

> Also, curious about why you decided to switch to using `WordPressComponentProps` only for `VisualLabel` and not for `BaseControl` in the same file?

No reason other than to keep the PR scope minimal, since there's already a lot going on!